### DISCUSSION TITLE
fail to switch master and log said Update master address from 192.168.47.129:10911 to null

### DISCUSSION THREAD

**Discussion by zuoHx580** [apache/rocketmq#6616] [Q&A]
version: 5.1
jdk:1.8
enviroment:
![image](https://user-images.githubusercontent.com/45234194/232956035-e8eb867c-faf1-4960-bf5b-ee85b4ae14c5.png)

and three nameserve nested DLedger Controller on  **192.168.47.128**,    **192.168.47.129**,    **192.168.47.130**

i shutdown the b-master（192.168.47.129）manually and then I found slave（192.168.47.131） can't become master

here is log:
![image](https://user-images.githubusercontent.com/45234194/232956658-b2907ac0-c8cf-4fed-ba8a-70ef707f3a62.png)

**Comment by RongtongJin** [comment]
@zuoHx580 是否还有其他日志，另外可以用getSyncStateSet命令查看下controller中的元数据情况？

**Reply by zuoHx580** [reply]
> @zuoHx580 是否还有其他日志，另外可以用getSyncStateSet命令查看下controller中的元数据情况？

解决问题了，原因时发现有个节点没配controller的地址😂
